iceoryx_examples/automotive_soa/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.16)
+project(example_automotive_soa)
+
+include(GNUInstallDirs)
+
+find_package(iceoryx_posh CONFIG REQUIRED)
+find_package(iceoryx_hoofs CONFIG REQUIRED)
+
+get_target_property(ICEORYX_CXX_STANDARD iceoryx_posh::iceoryx_posh CXX_STANDARD)
+include(IceoryxPlatform)
+
+# @todo #1332 enable 1:N communication cmake switch
+
+option(USE_UDS "Send Events via UNIX domain sockets" OFF)
+
+add_executable(iox-cpp-automotive-skeleton ./iox_automotive_skeleton.cpp ./minimal_skeleton.cpp)
+target_link_libraries(iox-cpp-automotive-skeleton
+    iceoryx_posh::iceoryx_posh
+)
+target_compile_options(iox-cpp-automotive-skeleton PRIVATE ${ICEORYX_WARNINGS} ${ICEORYX_SANITIZER_FLAGS})
+
+if(USE_UDS)
+  target_compile_definitions(iox-cpp-automotive-skeleton PUBLIC -DUSE_UDS)
+endif(USE_UDS)
+
+target_include_directories(iox-cpp-automotive-skeleton PRIVATE include)
+
+add_executable(iox-cpp-automotive-proxy ./iox_automotive_proxy.cpp ./minimal_proxy.cpp)
+target_link_libraries(iox-cpp-automotive-proxy
+    iceoryx_posh::iceoryx_posh
+)
+target_compile_options(iox-cpp-automotive-proxy PRIVATE ${ICEORYX_WARNINGS} ${ICEORYX_SANITIZER_FLAGS})
+
+if(USE_UDS)
+  target_compile_definitions(iox-cpp-automotive-proxy PUBLIC -DUSE_UDS)
+endif(USE_UDS)
+
+target_include_directories(iox-cpp-automotive-proxy PRIVATE include)
```

Comment:
@elBoberido 

3. Somehow this leads to  [this error](https://github.com/eclipse-iceoryx/iceoryx/runs/6393043116?check_suite_focus=true) only on the CI:

```
 [100%] Built target iox-cpp-automotive-proxy
Install the project...
-- Install configuration: ""
-- Installing: /home/runner/work/iceoryx/iceoryx/build/install/prefix/bin/iox-cpp-automotive-skeleton
-- Set runtime path of "/home/runner/work/iceoryx/iceoryx/build/install/prefix/bin/iox-cpp-automotive-skeleton" to ""
-- Installing: /home/runner/work/iceoryx/iceoryx/build/install/prefix/bin/iox-cpp-automotive-proxy
-- Set runtime path of "/home/runner/work/iceoryx/iceoryx/build/install/prefix/bin/iox-cpp-automotive-proxy" to ""
CMake Error: The source directory "/home/runner/work/iceoryx/iceoryx/iceoryx_examples/automotive_soa/include" does not appear to contain CMakeLists.txt.
```

Any clue why CMake is searching for a `.txt` in `include`? The `icediscovery` example is doing the same, but does not run into this issue.

Reply:
Solved!